app/views/katello/api/v2/packages/base.json.rabl

```diff
@@ -3,3 +3,6 @@ object @resource
 attributes :id, :uuid, :name, :version, :release, :arch, :epoch, :summary
 attributes :filename, :sourcerpm, :checksum
 attributes :nvrea, :nvra
+
+node(:hosts_available_count) { |m| m.hosts_available(params[:organization_id]).count }
+node(:hosts_applicable_count) { |m| m.hosts_applicable(params[:organization_id]).count }
```

Comment:
@jlsherrill wondering if this'll cause a hit to performance? we do it for errata as well.